### DISCUSSION TITLE
VM not detecting USB

### DISCUSSION THREAD

**Discussion by ghost** [utmapp/UTM#4421] [UTM Help (macOS)]
Hey, I think this has not been reported before, at least about the current versions of UTM. 

The thing is that suddenly my VM does not detect devices connected through USB ports. I had a Kali VM which detected them perfectly so the configuration of this one is the same as I have checked. In "input" the option to share from host is marked and my machine detects them so the hardware is good. 

Is anybody experiencing this problem? I already reinstalled UTM. Thanks to all beforehand. 

I have a mac mini m1

**Comment by osy** [comment]
Have you tried restarting your Mac?

**Reply by ghost** [reply]
Yeah, but still nothing

**Reply by gitblanc** [reply]
it happens the same to me :/, but sometimes it works

**Comment by ghost** [comment]
I found the issue. It was this option in the UTM exterior window. With others, there is an icon to activate the USB device.

Thanks you for your time and I sincerely hope there are a bunch of other idiots like me to whom this will be useful.

The time I wasted...

**Reply by osy** [reply]
Sorry can you explain what happened? What is the "UTM exterior window"? Is it possible to provide a screenshot of what you're referring to? Thanks

**Reply by ghost** [reply]
No problem. After initiating the VM, I have to enable my device there. Just activating this the problem is solved.

![1](https://user-images.githubusercontent.com/80861969/190654811-dfdb8ba4-7464-49ac-951a-a99e47a4f4cf.png)
![2](https://user-images.githubusercontent.com/80861969/190654825-4a86a098-4de4-49b9-97dc-e94f46a3b4b3.png)

**Reply by noobnoobdc137** [reply]
Hello I have the option but mine is disabled
How can I enable it?
My machine: Mac book pro (M2)
OS: MacOS Monterey (12.6)
OS on UTM:  MacOS Monterey (12.6)

<img width="204" alt="image" src="https://user-images.githubusercontent.com/55765209/194728157-e91f1eac-d139-4295-8d5c-05984d2902b1.png">

Thank you for the help

**Reply by noobnoobdc137** [reply]
> Hello I have the option but mine is disabled How can I enable it? My machine: Mac book pro (M2) OS: MacOS Monterey (12.6) OS on UTM: MacOS Monterey (12.6)
> 
> <img alt="image" width="204" src="https://user-images.githubusercontent.com/55765209/194728157-e91f1eac-d139-4295-8d5c-05984d2902b1.png">
> 
> Thank you for the help

Hello @Treintillin and @osy  can you help me on this. Thanks :)

**Reply by thanhBkap** [reply]
Thanks, you saved my day
